chaoskube/chaoskube_test.go

```diff
@@ -136,33 +140,38 @@ func (suite *Suite) TestCandidates() {
 	for _, tt := range []struct {
 		labelSelector      string
 		annotationSelector string
+		kindSelector       string
 		namespaceSelector  string
 		pods               []map[string]string
 	}{
-		{"", "", "", []map[string]string{foo, bar}},
-		{"app=foo", "", "", []map[string]string{foo}},
-		{"app!=foo", "", "", []map[string]string{bar}},
-		{"", "chaos=foo", "", []map[string]string{foo}},
-		{"", "chaos!=foo", "", []map[string]string{bar}},
-		{"", "", "default", []map[string]string{foo}},
-		{"", "", "default,testing", []map[string]string{foo, bar}},
-		{"", "", "!testing", []map[string]string{foo}},
-		{"", "", "!default,!testing", []map[string]string{}},
-		{"", "", "default,!testing", []map[string]string{foo}},
-		{"", "", "default,!default", []map[string]string{}},
+		{"", "", "", "", []map[string]string{foo, bar}},
+		{"app=foo", "", "", "", []map[string]string{foo}},
+		{"app!=foo", "", "", "", []map[string]string{bar}},
+		{"", "chaos=foo", "", "", []map[string]string{foo}},
+		{"", "chaos!=foo", "", "", []map[string]string{bar}},
+		{"", "", "", "default", []map[string]string{foo}},
+		{"", "", "", "default,testing", []map[string]string{foo, bar}},
+		{"", "", "", "!testing", []map[string]string{foo}},
+		{"", "", "", "!default,!testing", []map[string]string{}},
+		{"", "", "", "default,!testing", []map[string]string{foo}},
+		{"", "", "", "default,!default", []map[string]string{}},
```

Comment:
It looks like the `kindSelector` is never really tested here. Its value is always `""`.

Reply:
I removed that for now.